docs/howto/operate/grafana.md

```diff
@@ -80,7 +80,7 @@ The key you create needs admin permissions.
 
 Encrypt and store this key using `sops` in `secrets/config/clusters/<cluster>.yaml` under `grafana_token` key.
 
-This key will be used by the [`deploy-grafana-dashboards` workflow](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to deploy some default grafana dashboards for JupyterHub using [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards).
+This key will be used by the [`deploy-grafana-dashboards` workflow](https://github.com/2i2c-org/infrastructure/tree/HEAD/.github/workflows/deploy-grafana-dashboards.yaml) to deploy some default grafana dashboards for JupyterHub using [`jupyterhub/grafana-dashboards`](https://github.com/jupyterhub/grafana-dashboards).
```

Comment:
I'm a bit confused on the difference between blob and head. Both links work, but if you use `/tree/HEAD` you are redirected to `/blob/<commit hash>` while if you use `/blob/HEAD` you remain fixed and it works.

This LGTM as it is, no change needed, but off topic: do you know something about it?

Reply:
Hmmm, I updated the top comment with my thoughts. I wonder if `blob` is required for viewing files and `tree` required for viewing directories? Let's see what the linkcheck test says...

Reply:
> Hmmm, I updated the top comment with my thoughts. I wonder if blob is required for viewing files and tree required for viewing directories? Let's see what the linkcheck test says...

Ahh haha yeah, actually, the behavior i wrote about is reversed for directories. I guess they support blob/tree for both directories/files, but they do `blob` and `tree` depending on what kind of view they render (directory viewing kind of view for tree / file viewing kind of view for blob). If they switch from blob -> tree or tree -> blob, they pin the reference HEAD to a specific commit.

Reply:
So kinda doesn't matter, we get redirected to the right place anyway :D

Reply:
My summary becomes:

The choice is harmless, but you get another HTTP 301 redirect to follow using `/blob/HEAD` or `/tree/HEAD` when referencing the wrong associated type. `/tree` goes with directory views, `/blob` goes with file views.